packets.go

```diff
@@ -228,7 +233,22 @@ func (mc *mysqlConn) writeAuthPacket(cipher []byte) error {
 	// User Password
 	scrambleBuff := scramblePassword(cipher, []byte(mc.cfg.passwd))
 
+	attrs := make(map[string]string)
```

Comment:
It is completely unnecessary to build a `map[string]string` first. it's just a waste of memory and CPU time.
Furthermore it is not necessary to convert values to string first (e.g. os.Getpid()) every time.

Moreover those values don't change during runtime. So they could be read into a buffer in `init()` and then we could just copy this buffer here.